README.md

```diff
@@ -1,83 +1,115 @@
-[![PyPI Status Badge](https://badge.fury.io/py/armory-testbed.svg)](https://pypi.org/project/armory-testbed)
-[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/armory-testbed)](https://pypi.org/project/armory-testbed)
-[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-[![Docs](https://readthedocs.org/projects/armory/badge/)](https://armory.readthedocs.io/en/latest/)
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
+
+<!-- PROJECT LOGO !-->
+<br />
+<div align="center">
+  <a href="/">
+    <img src="/docs/assets/logo.png" width="50%" title="ARMORY Logo">
+  </a>
+
+  <p align="center">
+    Adversarial Robustness Evaluation Test Bed
+    <br />
+    <a href="/docs"><strong>🔍 Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="/../../issues">🐛 Report a Bug</a>
+    ·
+    <a href="/../../issues">📝 Request a Feature</a>
+  </p>
+
+</div>
+
+
 
 # Overview
 
-ARMORY is a test bed for running scalable evaluations of adversarial defenses. 
-Configuration files are used to launch local or cloud instances of the ARMORY docker 
-containers. Models, datasets, and evaluation scripts can be pulled from external 
-repositories or from the baselines within this project. 
+ARMORY is a test bed for running scalable evaluations of adversarial defenses.
+Configuration files are used to launch local or cloud instances of the ARMORY docker
+containers. Models, datasets, and evaluation scripts can be pulled from external
+repositories or from the baselines within this project.
 
-Our evaluations are created so that attacks and defenses may be 
-interchanged. To do this we standardize all attacks and defenses as subclasses of 
+Our evaluations are created so that attacks and defenses may be
+interchanged. To do this we standardize all attacks and defenses as subclasses of
 their respective implementations in the [Adversarial Robustness Toolbox (ART)](https://github.com/Trusted-AI/adversarial-robustness-toolbox) hosted by the LF AI & Data Foundation (LFAI).
 
 
 # Installation & Configuration
-``` 
+```bash
 pip install armory-testbed
 ```
 
-Upon installing armory, a directory will be created at `~/.armory`. This user 
-specific folder is the default directory for downloaded datasets, model weights, and 
-evaluation outputs. 
+Upon installing armory, a directory will be created at `~/.armory`. This user
+specific folder is the default directory for downloaded datasets, model weights, and
+evaluation outputs.
 
 To change these default directories simply run `armory configure` after installation.
 
 # Usage
 
-There are three ways to interact with the armory container system.
+There are three ways to interact with Armory's container system.
 
 1) `armory run`
```

Comment:
since we're here, "armory run" is not a `literal` it should be a `## armory run` subheading. Same goes for (2) and (3) below.